leveldb-library.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         =  'leveldb-library'
-  s.version      =  '1.22.3'
+  s.version      =  '1.23.0'
```

Comment:
Since 1.22.0 and 1.23.0 map to tags in https://github.com/google/leveldb, we should either make this 1.22.4 or switch to a meta-versioning scheme like  https://github.com/google/nanopb-podspec?tab=readme-ov-file#version-numbers

Reply:
Discussed offline. Going with 1.22.4